src/model/channel/mod.rs

```diff
@@ -425,6 +425,21 @@ impl Channel {
         }
     }
 
+    /// Retrieves the position of the inner [`GuildChannel`] or
+    /// [`ChannelCategory`].
+    ///
+    /// If other channel types are used it will return None.
+    ///
+    /// [`GuildChannel`]: struct.GuildChannel.html
+    /// [`CatagoryChannel`]: struct.ChannelCategory.html
+    pub fn position(&self) -> Option<i64> {
+        match self {
```

Comment:
To let the `1.25` travis job to build, you need to deference `self` here:
```suggestion
        match *self {
```